src/puppetlabs/puppetdb/middleware.clj

```diff
@@ -247,7 +248,7 @@
                       (doseq [metric-root metric-roots
                               :let [meter-key [:meters metric-root status]]
                               :when (not (get-in @storage meter-key))]
-                        (swap! storage assoc-in meter-key (meter [prefix metric-root (str status)] "reqs/s"))
```

Comment:
Not sure where this was surfaced before but note that we no longer specify `reqs/s` in the new `meter` interface.